deploy/helms/legion/templates/edge/edge-server.yaml

```diff
@@ -27,6 +27,8 @@ spec:
           value: "{{ .Release.Name }}-graphite.{{ .Release.Namespace }}.svc.{{ .Values.localDomain }}"
         - name: STATSD_PORT
           value: "8125"
+        - name: JWT_SECRET
```

Comment:
This should be done as K8S secret.

Reply:
@akharlamov It's required for the nginx-jwt module. It expects *JWT_SECRET* environment variable. 
https://github.com/auth0/nginx-jwt#configuration

Reply:
@aliaksandr-d yeah, you are right. But @akharlamov says that you have to 1) create K8S Secret 2) mount it as [env variable](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables)

Reply:
Fixed.